src/views/DashboardView.vue

```diff
@@ -30,16 +36,34 @@ watchEffect(async () => {
   }
 });
 
-const globalStats = ref({
-  totalTools: 2702,
-  toolsMissingInfo: 2701,
-  contributionsLast30Days: 73,
-  contributionsTotal: 226,
+const contributionsMetrics = ref({});
+watchEffect(async () => {
+  try {
+    contributionsMetrics.value = await getContributionsMetrics();
+  } catch (error) {
+    isError.value = true;
+    console.log(error);
+  }
 });
 
-const userStats = ref({
-  contributionsLast30Days: 4,
-  contributionsTotal: 10,
+const toolsMetrics = ref({});
+watchEffect(async () => {
+  try {
+    toolsMetrics.value = await getToolsMetrics();
+  } catch (error) {
+    isError.value = true;
+    console.log(error);
+  }
+});
+
+const userMetrics = ref({});
+watchEffect(async () => {
+  try {
+    userMetrics.value = await getUserMetrics();
+  } catch (error) {
+    isError.value = true;
+    console.log(error);
+  }
```

Comment:
Right now this will run and throw an error if there's no user logged in, and the lines will appear on the table without any numbers attached (because, of course, there are none).  I would put a check on this so that the page doesn't render the "user contributions" part of the table if no user is logged in.  (The same goes for the "My Contributions" table, though that's outside the scope of this PR.)

Reply:
Thank you for this.. I will make sure to add the checks for the two tables

Reply:
Great.  Now that #39 has gone through it should be as easy as `if (props.currentUser)`   I've already sorted the My Contributions table in that PR so there's nothing that needs to be done there.